File: website/public/javascripts/models/help-request.js
CBR.Models.HelpRequest = new Class({
    Extends: CBR.Models.JsonSerializableModel,

    options: {  // Defaults
    },

    getId: function () {
        return this.options.id;
    },

    getRequesterId: function () {
        return this.options.requesterId;
    },

    getTitle: function () {
        return this.options.title;
    },

    getDescription: function () {
        return this.options.description;
    },

    getCreationDatetime: function () {
        return this.options.creationDatetime;
    },

    getExpiryDate: function () {
        return this.options.expiryDate;
    }
});

File: selenium-tests/src/main/java/models/HelpRequest.java
package models;

public class HelpRequest {
    private String title;
    private String description;
    private String expiryDate;

    public HelpRequest(String title,
                       String description,
                       String expiryDate) {
        this.title = title;
        this.description = description;
        this.expiryDate = expiryDate;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getExpiryDate() {
        return expiryDate;
    }

    public void setExpiryDate(String expiryDate) {
        this.expiryDate = expiryDate;
    }
}
